src/test/scala/edu/ie3/simona/agent/em/EmAgentIT.scala

```diff
@@ -300,8 +300,8 @@ class EmAgentIT
           7200,
           weatherService.ref.toClassic,
           WeatherData(
-            WattsPerSquareMeter(50d),
-            WattsPerSquareMeter(150d),
+            WattsPerSquareMeter(45d),
+            WattsPerSquareMeter(140d),
```

Comment:
Whats the reason to change the weather here? (I would prefer not to change if not necessary, since there are more drafted PRs that would require an update...

Reply:
See the description of the PR